packages/website/src/docs/language-basics/models.md

```diff
@@ -4,3 +4,150 @@ title: Models
 ---
 
 # Models
+
+Cadl models are used to describe data shapes or schemas.
+
+## Model kinds
+
+Models can be used to represent 3 types:
+
+- [Primitives](#primitives)
+- [Record](#record)
+- [Array](#array)
+
+### Primitives
+
+These are types without any fields(For example `string`, `int32`, `boolean`, etc.)
+
+### Record
+
+Model records are structure with named fields called properties.
```

Comment:
I find `model records` to be confusing personally. Record models perhaps? Or just records?